### PR TITLE
LPS-181575 Modify label and icon for FDSViews edit link

### PR DESCRIPTION
**Pull request by edalgrin**
This PR modifies the label and the icon of the FDS Entry and View, edit and rename link

image
image

### CODE REVIEW

**Comment by edalgrin** [comment]
ci:test:relevant

**Comment by JerryAdrianoo** [comment]
Hey @edalgrin these failures are not related with your changes, except for those who need testfix.
I'll work on it and send the push.

**Comment by JerryAdrianoo** [comment]
Hey, @edalgrin Can you rebase this branch ? I need to run relevant to see if these fixes are ok.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx [hunk 1]
```diff
@@ -1020,15 +1020,23 @@ const FDSEntries = ({
...
+						separator: true,
+					},
+					{
+						icon: 'blank',
 						label: Liferay.Language.get('rename'),
 						onClick: onRenameClick,
 					},
+					{
+						type: "group",
+						separator: true,
```
Comment (thektan) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:1038]: I'm surprised SF didn't complain about this, but can we alphabetically sort `separator` and `type`?

**Review by thektan — CHANGES_REQUESTED** [review]
Hey @edalgrin I just noticed a small formatting thing if you could change while rebasing this. Thanks!

**Comment by edalgrin** [comment]
I'm on it, thanks!

**Comment by edalgrin** [comment]
ci:test:relevant

**Comment by JerryAdrianoo** [comment]
ci:test:relevant

**Comment by JerryAdrianoo** [comment]
LGTM!!

**Comment by JerryAdrianoo** [comment]
ci:test:relevant

**Comment by JerryAdrianoo** [comment]
The failures seem to be unrelated to the PR according to the [comparison](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1692360631037&testrayRunIdA=2407585495&testrayRunIdB=2408595034).
LGTM!

**Comment by edalgrin** [comment]
ci:test:sf

**Comment by edalgrin** [comment]
ci:test:sf

**Comment by edalgrin** [comment]
ci:forward:force

**Comment by edalgrin** [comment]
ci:stop

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx [hunk 2]
```diff
@@ -1020,15 +1020,23 @@ const FDSEntries = ({
 						data: {
 							id: 'view',
 						},
-						icon: 'view',
-						label: Liferay.Language.get('view'),
+						icon: 'pencil',
+						label: Liferay.Language.get('edit'),
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:1024]: To be consistent in code, can we change all `view*` in file into `*edit`:
- `onViewClick` -> `onEditClick`
- `getViewURL` -> `getEditURL`
- `id: 'view'` -> `id: 'edit'`, along with `actionId: 'edit'`,

**Comment by edalgrin** [comment]
hey @markocikos since these are optimization I'll create a follow-up PR for those change and avoid waiting for the CI anymore

**Comment by edalgrin** [comment]
ci:stop

**Comment by edalgrin** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/3612

**Comment by markocikos** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:test:stable

**Comment by JerryAdrianoo** [comment]
I'm on it!

**Comment by JerryAdrianoo** [comment]
Hey @edalgrin this failure is not related with your changes!
LGMT!

**Comment by thektan** [comment]
Manually forwarded this to BChan: https://github.com/brianchandotcom/liferay-portal/pull/139761

**Comment by thektan** [comment]
ci:close

**Pull request closed without merging**